src/components/application_manager/src/commands/hmi/vr_is_ready_response.cc

```diff
@@ -49,9 +49,15 @@ void VRIsReadyResponse::Run() {
   if (object[strings::msg_params].keyExists(strings::available)) {
     is_available = object[strings::msg_params][strings::available].asBool();
   }
-
   HMICapabilities& hmi_capabilities = application_manager_.hmi_capabilities();
   hmi_capabilities.set_is_vr_cooperating(is_available);
+
+  const HmiInterfaces::InterfaceState inteface_state =
+      is_available ? HmiInterfaces::STATE_AVAILABLE
+                   : HmiInterfaces::STATE_NOT_AVAILABLE;
+  HmiInterfaces& hmi_interfaces = application_manager_.hmi_interfaces();
+  hmi_interfaces.SetInterfaceState(HmiInterfaces::HMI_INTERFACE_VR,
+                                  inteface_state);
```

Comment:
@pestOO I think It should be done in scope of implementing other related CRQ, to clarify is it going to be correct for each one.